service/src/main/java/edu/tamu/catalog/service/FolioCatalogService.java

```diff
@@ -430,6 +426,33 @@ private void buildCoreItem(String instanceId, String barcode, NodeList nodes, Ma
         catalogItems.put(barcode, itemData);
     }
 
+    private LoanItem buildLoanItem(JsonNode loanJson) throws ParseException {
+        Date loanDate = loanJson.has("loanDate") ? folioDateToDate(loanJson.get("loanDate").asText()) : null;
+        Date loanDueDate = loanJson.has("dueDate") ? folioDateToDate(loanJson.get("dueDate").asText()) : null;
+        String overDueString = getNodeValue(loanJson, "overdue");
+        boolean overDue = false;
```

Comment:
```suggestion
        boolean overDue = overDueString == null ? false : Boolean.valueOf(overDueString);
```

Reply:
We can't use this suggestion as it does the same logic below. But I am ok with this change.